handlers/handlers.go

```diff
@@ -1,33 +1,51 @@
 package handlers
 
 import (
+	"api/entities"
 	"encoding/csv"
 	"encoding/json"
-	"fmt"
 	"net/http"
 	"os"
 	"strconv"
 
 	"github.com/gorilla/mux"
 )
 
+const FilePath = "assets/pokemon.csv"
```

Comment:
This could go in a configuration file, then you could use a package, such as [viper](https://github.com/spf13/viper) to read the values.